int32 agentID
int32 x
int32 y
int32 z
float64 theta

---
sensor_msgs/PointCloud pointcloud